app/celery/scheduled_tasks.py

```diff
@@ -137,30 +143,30 @@ def check_job_status():
     from jobs
     where job_status == 'in progress'
     and template_type in ('sms', 'email')
-    and scheduled_at or created_at is older that 120 minutes.
+    and scheduled_at or created_at is older than 60 minutes.
     if any results then
         raise error
         process the rows in the csv that are missing (in another task) just do the check here.
     """
-    minutes_ago_120 = datetime.utcnow() - timedelta(minutes=120)
-    minutes_ago_125 = datetime.utcnow() - timedelta(minutes=125)
+    minutes_ago_60 = datetime.utcnow() - timedelta(minutes=60)
```

Comment:
I reverted this from 120 minutes back to 60. Originally this was 30 minutes and we bumped it up to [60](https://github.com/cds-snc/notification-api/commit/db955f3038742b641af5997cd2a1ed1e96bd9520) then to [120](https://github.com/cds-snc/notification-api/commit/6f71d29a8c60dcadee9562ea624d0c34171a41e9) as a bandaid fix for the issue this PR solves. Perhaps at some point we could revert back to 30 minutes as well?

Reply:
We could do 30 minutes now to eagerly fail: if this is deemed to fail even with the new fix, we'd know sooner than later.

Reply:
Reverted back to 30 minutes!